packages/block-editor/src/components/block-navigation/style.scss

```diff
@@ -274,15 +280,70 @@
 	}
 }
 
+// Chevron container metrics.
+.block-editor-block-navigation__expander {
+	height: $icon-size;
+	margin-left: $grid-unit-05;
+	width: $icon-size;
+}
+
 // First level of indentation is aria-level 2, max indent is 8.
 // Indent is a full icon size, plus 4px which optically aligns child icons to the text label above.
 $block-navigation-max-indent: 8;
-.block-editor-block-navigation-leaf[aria-level] .block-editor-block-icon {
-	margin-left: ( $grid-unit-30 + $grid-unit-05 ) * $block-navigation-max-indent;
+.block-editor-block-navigation-leaf[aria-level] .block-editor-block-navigation__expander {
+	margin-left: ( $grid-unit-30 ) * $block-navigation-max-indent;
 }
-@for $i from 0 through $block-navigation-max-indent {
-	.block-editor-block-navigation-leaf[aria-level="#{ $i + 1 }"] .block-editor-block-icon {
-		margin-left: ( $grid-unit-30 + $grid-unit-05 ) * $i;
+
+.block-editor-block-navigation-leaf:not([aria-level="1"]) {
+	.block-editor-block-navigation__expander {
+		margin-right: 4px;
 	}
 }
 
+.block-editor-block-navigation-leaf[aria-level="1"] .block-editor-block-navigation__expander {
+	margin-left: 0;
+}
+
+.block-editor-block-navigation-leaf[aria-level="2"] .block-editor-block-navigation__expander {
+	margin-left: 24px;
```

Comment:
Is there a formula for the pixel indentation?

Reply:
Thanks! @jasmussen @jameskoster I updated this in https://github.com/WordPress/gutenberg/pull/32117/commits/7fab9c8e484c081112f9cd98a357653b2e69c4c6 Let me know if that's too confusing, and we can keep the expanded form otherwise.

<img width="1751" alt="Screen Shot 2021-06-11 at 9 50 14 AM" src="https://user-images.githubusercontent.com/1270189/121722862-b7a8cf00-ca9a-11eb-8530-6515ba408ef6.png">

Reply:
Looks good to me! Thanks.